The types are now broken

Since new major release the types are broken. Try to install this package in separate project.

`Cannot find module '@zitadel/node/credentials' or its corresponding type declarations`

Types work if using whole path `@zitadel/node/dist/esm/credentials`, but since this path is not in `exports`, node does not want to work with that

--- a/package.json
+++ b/package.json
@@ -63,5 +63,15 @@
       "./credentials": "./src/credentials/index.ts",
       "./package.json": "./package.json"
     }
+  },
+  "typesVersions": {
+    "*": {
+      "api": [
+        "dist/commonjs/api/index.d.ts"
+      ],
+      "credentials": [
+        "dist/commonjs/credentials/index.d.ts"
+      ]
+    }
   }
 }

